src/libaktualizr/config/config.cc

```diff
@@ -276,21 +278,34 @@ void Config::postUpdateValues() {
 
   checkLegacyVersion();
   initLegacySecondaries();
+  LOG_TRACE << "Final configuration that will be used: \n" << (*this);
 }
 
-void Config::updateFromDirs() {
-  for (const auto& dir : config_dirs_) {
-    for (const auto& config : Utils::glob((dir / "*.conf").string())) {
-      updateFromToml(config);
+void Config::updateFromDirs(const std::vector<boost::filesystem::path>& configs) {
+  std::map<std::string, boost::filesystem::path> configs_map;
+  for (const auto& config : configs) {
+    if (boost::filesystem::is_directory(config)) {
+      for (const auto& config_file : Utils::glob((config / "*.conf").string())) {
```

Comment:
Shouldn't this be finding files with a `.toml` extension? That's what all of our current config files use.